dev-tools/phpstan/baseline/_loader.php

```diff
@@ -1,6 +1,5 @@
 <?php declare(strict_types = 1);
 
-// total 482 errors
```

Comment:
While I like these numbers, they create annoying git conflicts when doing multiple pull requests changing the baseline.
So I think it is not worth it to keep these numbers.